molecule_openstack/cookiecutter/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/molecule.yml

```diff
@@ -0,0 +1,16 @@
+---
+dependency:
+  name: {{ cookiecutter.dependency_name }}
+driver:
+  name: {{ cookiecutter.driver_name }}
+platforms:
+  - name: molecule-{{ cookiecutter.role_name }}
+    image: ${MOLECULE_OPENSTACK_IMAGE}
+    flavor: ${MOLECULE_OPENSTACK_FLAVOR}
```

Comment:
To keep current code behavior you need something like `${MOLECULE_OPENSTACK_NETWORK:molecule}`, and similar for ssh_user. If you do not do this, you will force people to define these extra env-vars.

Reply:
fixed